codecs/avif/Makefile

```diff
@@ -36,7 +38,8 @@ HELPER_MAKEFLAGS := -f helper.Makefile
 all: $(OUT_ENC_JS) $(OUT_DEC_JS) $(OUT_ENC_MT_JS)
 
 # ST-Encoding
-$(OUT_ENC_JS): $(OUT_ENC_CPP) $(CODEC_DIR)/CMakeLists.txt $(LIBAOM_DIR)/CMakeLists.txt $(LIBSHARPYUV)
+$(OUT_ENC_JS): $(OUT_ENC_CPP) $(CODEC_DIR)/CMakeLists.txt $(LIBAOM_DIR)/CMakeLists.txt $(LIBSHARPYUV_ST)
+	mkdir -p $(LIBWEBP_DIR)/build && cp $(LIBSHARPYUV_ST) $(LIBSHARPYUV)
```

Comment:
Due to the CMakesList.txt in `libavif` the `AVIF_LOCAL_LIBSHARPYUV` has to be located at `/ext/libwebp/build/libsharpyuv.a`

The only workaround I could think of was to copy the relevant file over for each build. Does anyone have a cleaner idea? Or would it be worth requesting an improvement upstream to provide a custom location?

Reply:
@RReverser No idea if you have time for this, but if you do: Any thoughts on this?

Reply:
Hm looking at their CMake, not really, copy or symlink seems like the best option. Often build configs allow to specify custom paths to their dependencies, but apparently not this one. https://chromium.googlesource.com/external/github.com/AOMediaCodec/libavif/+/refs/heads/chromium-m119/CMakeLists.txt#216